src/irmin-pack/unix/dispatcher.ml

```diff
@@ -0,0 +1,255 @@
+open Import
+module Payload = Control_file.Latest_payload
```

Comment:
I am not super convinced by the name of that component. The fact that it dispatches behind the scene behind a prefix and a suffix file is mostly hidden from the user (only the GC needs to know a few more internal stuff). I would suggest calling it "pack" something-something (and if needed rename the other "pack" file to prefix or whatever :p)